website/docs/en/config/experiments.mdx

```diff
@@ -353,9 +353,17 @@ export default {
   experiments: {
     parallelCodeSplitting: true,
   },
+  optimization: {
+    removeAvailableModules: true,
+  },
 };
 ```
 
+:::warning
+When `parallelCodeSplitting` is enabled, make sure to also enable `optimization.removeAvailableModules` (this is also enabled by default since version 1.3.0).
+This ensures consistency with the old code splitting algorithm, which enforced `removeAvailableModules` internally and ignored the `optimization.removeAvailableModules` configuration.
```

Comment:
```suggestion
When `parallelCodeSplitting` is enabled, ensure that 'optimization.removeAvailableModules' is also enabled (this has been enabled by default since version 1.3.0).

This maintains consistency with the previous code splitting algorithm, which enforced `removeAvailableModules` internally and ignored the `optimization.removeAvailableModules` configuration.
```